redaxo/src/addons/debug/boot.php

```diff
@@ -114,6 +114,36 @@
 
     $ep->table('Executed Extension Points', rex_extension_debug::getExtensionPoints());
     $ep->table('Registered Extensions', rex_extension_debug::getExtensions());
-
-    $clockwork->resolveRequest()->storeRequest();
-});
+};
+
+$console = rex::getConsole();
+if ($console) {
+    rex_extension::register(rex_extension_point_console_shutdown::NAME, static function (rex_extension_point_console_shutdown $extensionPoint) use ($shutdownFn) {
+        $shutdownFn();
+
+        $command = $extensionPoint->getCommand();
+        $input = $extensionPoint->getInput();
+        $output = $extensionPoint->getOutput();
+        $exitCode = $extensionPoint->getExitCode();
+
+        $clockwork = rex_debug_clockwork::getInstance();
+        $clockwork
+            ->resolveAsCommand(
+                $command->getName(),
+                $exitCode,
+                $input->getArguments(),
+                $input->getOptions(),
+                $command->getDefinition()->getArgumentDefaults(),
```

Comment:
Brauchen wir hier ein array-diff?

https://gist.github.com/itsgoingd/bc7ae61d2272672a4de42dff7c37b518#file-clockworkconsolesupport-php-L53

Reply:
da Clockwork zwischen den gesetzten und den Defaults unterscheidet würde ichs machen. Ich schau mir das in der UI an, was "besser" ist.

Reply:
Der Unterschied ist nur in der Request liste sichtbar. Der ändert dort nur den Hovertext. Da aber das die wirklichen angehängten Args entspricht würde ich es mit `array_diff` machen.

ohne array-diff:
![image](https://user-images.githubusercontent.com/21292755/82729139-80cc1380-9cf5-11ea-8862-53148d3e2818.png)

mit array-diff:
![image](https://user-images.githubusercontent.com/21292755/82729149-8cb7d580-9cf5-11ea-851d-aaf000856488.png)

Passe ich an.